nautobot_device_lifecycle_mgmt/forms.py

```diff
@@ -985,6 +985,7 @@ class CVELCMForm(
     """CVE Lifecycle Management creation/edit form."""
 
     published_date = forms.DateField(widget=DatePicker())
```

Comment:
DateField() is not compatible with Nautobot 1.3.3

Need to refactor to use a field compatible with both 1.1.6 and 1.3.3

Reply:
What's the issue with 1.3?

Reply:
My deepest apologies, my comment was on the wrong code.  Resolving this.